webboot/webboot.go

```diff
@@ -8,9 +8,24 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"log"
+
+	"github.com/u-root/webboot/pkg/dhclient"
+)
+
+var (
+	ifName  = flag.String("InterfaceName", "^e.*", "Name of the interface")
```

Comment:
There's an undocumented convention where flags are lowercase and short. Perhaps, name this flag "interface" rather than "InterfaceName".

Reply:
Oh, sorry I did not realize that. Thank you for pointing it out. I will make the change.